gnomad/utils/sparse_mt.py

```diff
@@ -563,7 +563,8 @@ def default_compute_info(
         aggregations. This method can be used in cases where genotype data doesn't
         contain allele-specific annotations to approximate allele-specific annotations.
         Default is True.
-    :param n_partitions: Number of desired partitions for output Table. Default is 5000.
+    :param n_partitions: Optional number of desired partitions for output Table. If
+        None, naive_coalesce is performed. Default is 5000.
```

Comment:
```suggestion
        specified, naive_coalesce is performed. Default is 5000.
```
or `If None, naive_coalesce is skipped`?